src/main/ngapp/accio/src/app/editor/magic-wand.service.ts

```diff
@@ -175,4 +201,130 @@ export class MagicWandService {
 
     return invertedMask;
   }
+
+
+  /* Smarter flood fill tool */
+
```

Comment:
Update comment about what makes it smarter.

Reply:
Also I am not sure what scribbles is.  How is that different than xCoord and yCoord?

Reply:
Scribbles is a set of pixels, whose colors are used to expand the reference pixels that the tolerance logic uses to determine if a pixel should be part of the mask or not. The range of our tolerance is essentially being expanded to include colors within the scribbles set.
I've updated this to be more clear in the in-line function documentation.